x-pack/plugins/security_solution_serverless/public/get_started/card_step/step_complete_button.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React, { useCallback } from 'react';
+import icon_cross from '../images/icon_cross.svg';
+import { MARK_AS_DONE_TITLE, UNDO_MARK_AS_DONE_TITLE } from '../translations';
+
+const StepCompleteButtonComponent = ({
+  isStepDone,
+  onClick,
+}: {
+  isStepDone: boolean;
+  onClick: () => void;
+}) => {
+  const { euiTheme } = useEuiTheme();
+  const handleStepButtonClicked = useCallback(
+    (e) => {
+      e.preventDefault();
```

Comment:
NIT: if it's not a link with `href` defined there's no default action to prevent.